### PR TITLE
fix(packages): Fix the search page on edit package

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: server/apps/archive/user_content.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class UserContentResource(Resource):
     datasource = {
         'source': 'archive',
         'aggregations': aggregations,
-        'elastic_filter': {'not': {'exists': {'field': 'task.stage'}}}  # eve-elastic specific filter
+        'elastic_filter': {'not': {'exists': {'field': 'task.desk'}}}  # eve-elastic specific filter
```
Comment (amagdas) [inline server/apps/archive/user_content.py]: Why is this needed?

**Comment by amagdas** [comment]
:+1:

File: server/apps/archive/user_content.py (hunk 1 above)
Comment (ioanpocol) [inline server/apps/archive/user_content.py]: On personal we have to filter content created by current user and not assigned to any desk.

Comment (amagdas) [inline server/apps/archive/user_content.py]: And the previous filter didn't work?

Comment (ioanpocol) [inline server/apps/archive/user_content.py]: No

Comment (amagdas) [inline server/apps/archive/user_content.py]: :+1:

**Comment by petrjasek** [comment]
:+1:
